### PR TITLE
:running: build KB tools for k8s 1.17

### PR DESCRIPTION
**Pull request by mengqiy**
When this PR merges, it should trigger the cloud build pipeline.

### CODE REVIEW

File: build/thirdparty/darwin/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # - kubectl (fetch)
 # - etcd (fetch)
 
-FROM golang:1.12.14 as darwin
+FROM golang:1.14.7 as darwin
```
[the remark this answers is not in the document]
Comment (mengqiy) [reply build/thirdparty/darwin/Dockerfile:20]: There's no go.mod in this branch: https://github.com/kubernetes-sigs/kubebuilder/tree/tools-releases

**Comment by JoelSpeed** [comment]
Changes look good to me, thanks! Do you know what we can do about the owners file? 

Can we make a 1.18 release after this one has been done as well?

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

👍

**Comment by vincepri** [comment]
@DirectXMan12 Has powers to merge manually :)

**Comment by mengqiy** [comment]
>Do you know what we can do about the owners file?

I guess we can copy over the OWNERS file from the master branch to this branch.
Thought the copy in this branch may be out of sync from the one in master branch over time, it should not matter much :)

> Can we make a 1.18 release after this one has been done as well?

@JoelSpeed After we confirming that the tools for 1.17 is built correctly, feel free to create a PR to build for 1.18.

**Comment by vincepri** [comment]
@mengqiy Please add me as well to this file if possible

**Comment by mengqiy** [comment]
It seems the tools for 1.17 are built and published: https://storage.googleapis.com/kubebuilder-tools

> Please add me as well to this file if possible

Sure

**Pull request merged**